hardware/Netatmo.cpp

```diff
@@ -488,14 +526,146 @@ uint64_t CNetatmo::convert_mac(std::string mac)
 }
 
 
+uint64_t getValueFromString(const std::string& value)
+{
+	uint64_t result;
+	std::istringstream iss( value );
+	iss >> result;
+	return result;
+}
+
+std::string getStringFromValue(const uint64_t value)
```

Comment:
Instead of this function, please use **std::to_string**

Reply:
OK

Reply:
Done